### ISSUE TITLE
FTPS source with CP 6.0 is failing

### ISSUE THREAD

**Issue by vdesabou** [vdesabou/kafka-docker-playground#90]
```
[2020-09-28 06:23:12,268] ERROR Failed to start task ftps-source-json-0 (org.apache.kafka.connect.runtime.Worker)
io.confluent.common.config.ConfigException: Invalid value  for configuration schema.registry.ssl.engine.factory.class: Class  could not be found.
        at io.confluent.common.config.ConfigDef.parseType(ConfigDef.java:392)
        at io.confluent.common.config.ConfigDef.define(ConfigDef.java:110)
        at io.confluent.common.config.ConfigDef.define(ConfigDef.java:154)
        at io.confluent.kafka.schemaregistry.client.SchemaRegistryClientConfig.withClientSslSupport(SchemaRegistryClientConfig.java:44)
        at io.confluent.kafka.serializers.AbstractKafkaAvroSerDeConfig.baseConfigDef(AbstractKafkaAvroSerDeConfig.java:155)
        at io.confluent.connect.avro.AvroConverterConfig.<init>(AvroConverterConfig.java:27)
        at io.confluent.connect.avro.AvroConverter.configure(AvroConverter.java:61)
        at org.apache.kafka.connect.runtime.isolation.Plugins.newConverter(Plugins.java:298)
        at org.apache.kafka.connect.runtime.Worker.startTask(Worker.java:544)
        at org.apache.kafka.connect.runtime.distributed.DistributedHerder.startTask(DistributedHerder.java:1251)
        at org.apache.kafka.connect.runtime.distributed.DistributedHerder.access$1700(DistributedHerder.java:127)
        at org.apache.kafka.connect.runtime.distributed.DistributedHerder$10.call(DistributedHerder.java:1266)
        at org.apache.kafka.connect.runtime.distributed.DistributedHerder$10.call(DistributedHerder.java:1262)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by vdesabou** [comment]
Also seeing same error message with GCS Sink

```
[2020-09-29 08:44:54,061] INFO Instantiated task gcs-rbac-sink-0 with version 5.5.2 of type io.confluent.connect.gcs.GcsSinkTask (org.apache.kafka.connect.runtime.Worker)
[2020-09-29 08:44:54,071] ERROR Failed to start task gcs-rbac-sink-0 (org.apache.kafka.connect.runtime.Worker)
io.confluent.common.config.ConfigException: Invalid value  for configuration schema.registry.ssl.engine.factory.class: Class  could not be found.
        at io.confluent.common.config.ConfigDef.parseType(ConfigDef.java:392)
        at io.confluent.common.config.ConfigDef.define(ConfigDef.java:110)
        at io.confluent.common.config.ConfigDef.define(ConfigDef.java:154)
        at io.confluent.kafka.schemaregistry.client.SchemaRegistryClientConfig.withClientSslSupport(SchemaRegistryClientConfig.java:44)
        at io.confluent.kafka.serializers.AbstractKafkaAvroSerDeConfig.baseConfigDef(AbstractKafkaAvroSerDeConfig.java:155)
        at io.confluent.connect.avro.AvroConverterConfig.<init>(AvroConverterConfig.java:27)
        at io.confluent.connect.avro.AvroConverter.configure(AvroConverter.java:61)
        at org.apache.kafka.connect.runtime.isolation.Plugins.newConverter(Plugins.java:298)
        at org.apache.kafka.connect.runtime.Worker.startTask(Worker.java:546)
        at org.apache.kafka.connect.runtime.distributed.DistributedHerder.startTask(DistributedHerder.java:1251)
        at org.apache.kafka.connect.runtime.distributed.DistributedHerder.access$1700(DistributedHerder.java:127)
        at org.apache.kafka.connect.runtime.distributed.DistributedHerder$10.call(DistributedHerder.java:1266)
        at org.apache.kafka.connect.runtime.distributed.DistributedHerder$10.call(DistributedHerder.java:1262)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:834)
[2020-09-29 08:44:54,076] INFO [Worker clientId=connect-1, groupId=connect-cluster] Finished starting connectors and tasks (org.apache.kafka.connect.runtime.distributed.DistributedHerder)
```

**Comment by vdesabou** [comment]
Fixed with FTPS 1.0.3-preview
